File: Java/Processor.java
public class Processor
{
    // Processor attributes.
    private String name;
    private int price;

    /* Constructor. */
    
    // Empty constructor.
    public Processor()
    {

    }

    // Constructor with processor's data.
    public Processor(String name, int price)
    {
        this.name = name;
        this.price = price;
    }

    /* Setter and Getter. */

    // Set name attribute.
    public void setName(String name)
    {
        this.name = name;
    }

    // Set price attribute.
    public void setPrice(int price)
    {
        this.price = price;
    }

    // Get name attribute.
    public String getName()
    {
        return this.name;
    }

    // Set price attribute.
    public int getPrice()
    {
        return this.price;
    }
}

File: Python/Processor.py
class Processor:

    # Processor attributes.
    __name = ""
    __price = 0

    # Constructor.
    # Please take note that after a few try, Python seems to not accept
    # multiple constructor. So I decided to set it as data constructor. #

    # Constructor with processor's data.
    def __init__(self, name, price):
        self.__name = name
        self.__price = price
    
    # Setter and Getter. #

    # Set name attribute.
    def setName(self, name):
        self.__name = name
    
    # Set price attribute.
    def setPrice(self, price):
        self.__price = price

    # Get name attribute.
    def getName(self):
        return self.__name
    
    # Get price attribute.
    def getPrice(self):
        return self.__price